dspace-api/src/main/java/org/dspace/xmlworkflow/storedcomponents/ClaimedTask.java

```diff
@@ -115,4 +123,29 @@ public void setWorkflowID(String workflowID) {
     public String getWorkflowID() {
         return workflowId;
     }
+
+    @Override
+    public String getTypeText() {
+        return "claimedtask";
```

Comment:
Should this be in `Constants`?

Reply:
done https://github.com/DSpace/DSpace/pull/2312/files#diff-8a2f0d69cfbc625b36f5d0da020e399bR129